setup.py

```diff
@@ -12,10 +12,10 @@
     author_email="CurtLHampton@gmail.com",
     url="https://github.com/Workday/prism-python",
     packages=["prism"],
-    package_data={"prism": ["data/*"]},
-    entry_points={"console_scripts": ["prism=prism.cli:main"]},
+    package_data={"prism": ["data/*", "commands/*"]},
```

Comment:
I don't believe we need to include `"commands/*"` here, as `package_data` is meant to tell setuptools to include the data files in this directory when building the package.  This is necessary to the data test files are included when the package builds through the CI platform.

Reply:
When I don't include the directory, I get errors on when I run: 

[ec2-user@ip-172-31-26-32 ~]$ prism tables get
Traceback (most recent call last):
  File "/home/ec2-user/.local/bin/prism", line 5, in <module>
    from prism.cli import cli
  File "/home/ec2-user/.local/lib/python3.9/site-packages/prism/cli.py", line 9, in <module>
    from .commands import tables_commands as t_commands
**ModuleNotFoundError: No module named 'prism.commands'**

Reply:
Check out this [documentation](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#package-data).  I don't believe `pacakge_data` is the right way to solve that issue. 

> Often, additional files need to be installed into a [package](https://packaging.python.org/en/latest/glossary/#term-Import-Package). These files are often data that’s closely related to the package’s implementation, or text files containing documentation that might be of interest to programmers using the package. These files are called “package data”.

Instead, I think we need to explicitly tell the package where `prism.commands` is within the `__init__.py` file.  Check out this [documentation](https://docs.python.org/3.8/tutorial/modules.html#packages) for an example.